inscriptions/ISic000647.xml

```diff
@@ -232,28 +265,45 @@
                     </bibl>
                     <bibl type="corpus" n="IMusCatania">
                         <citedRange>212</citedRange>
-                        <ptr target="https://www.zotero.org/groups/382445/items/2EUC82DH"/>
+                        <ptr target="https://zotero.org/groups/382445/items/2EUC82DH"/>
                     </bibl>
                     <bibl>
                         <author>Ferrua</author>
                         <date>1989</date>
                         <citedRange>no.426</citedRange>
-                        <ptr target="https://www.zotero.org/groups/382445/items/759ZXHKQ"/>
+                        <ptr target="https://zotero.org/groups/382445/items/759ZXHKQ"/>
                     </bibl>
                     <bibl>
                         <author>Libertini</author>
                         <date>1930</date>
                         <citedRange>317 no.3</citedRange>
-                        <ptr target="https://www.zotero.org/groups/382445/items/SKUU2R8X"/>
+                        <ptr target="https://zotero.org/groups/382445/items/SKUU2R8X"/>
                     </bibl>
                     <bibl>
                         <author>Manganaro</author>
                         <date>1994</date>
                         <citedRange>88-89 no.6 fig.9</citedRange>
-                        <ptr target="https://www.zotero.org/groups/382445/items/532NF7SV"/>
+                        <ptr target="https://zotero.org/groups/382445/items/532NF7SV"/>
+                    </bibl>
+                </listBibl>
+                <listBibl type="discussion">
+                    <bibl type="corpus" n="IMusCatania">
```

Comment:
not clear what the second ref to IMusCatania is here in discussion? page number? corpus number? something distinct from the entry for no. 212?